scripts/entity-generation/2-create-entity-types.ts

```diff
@@ -152,26 +152,6 @@ for (const { dir, config } of configs) {
 
 const results = resolveEntityTypeJoins(intermediateResults, args.values['force-generate-joins']);
 
-for (const {
-  entityType,
-  metadata: { domain, module },
-} of results) {
-  await mkdir(path.resolve(args.values.out, 'entity-types', domain, module), { recursive: true });
-  await write('entity-types', domain, module, `${entityType.name}.json5`, json5.stringify(entityType, null, 2));
-  await write(
-    'csv',
-    domain,
-    module,
-    `${entityType.name}.csv`,
-    await entityTypeToCsv(
-      entityType,
-      memoize((searchId) =>
-        Promise.resolve(results.map((r) => r.entityType).find((et) => et.id === searchId) ?? ({} as EntityType)),
-      ),
-    ),
-  );
-}
-
```

Comment:
moved to after translation handling so that translations can be included in csv output, too